client/settings/digital-wallets/test/digital-wallets.test.js

```diff
@@ -0,0 +1,82 @@
+/** @format **/
```

Comment:
A very minor thing but this file should probably be called `index.js`.

Reply:
Sorry, force of habit, renamed 👍